client/src/components/DecisionButtons.vue

```diff
@@ -16,13 +21,45 @@ export default {
     return {
       warnDecision: false,
       newComment: '',
+      confirmedPresent: [],
+      confirmedAbsent: [],
     };
   },
   computed: {
+    ...mapState([
+      'currentViewData',
+    ]),
     ...mapGetters([
       'currentViewData',
       'myCurrentProjectRoles',
     ]),
+    artifacts() {
+      return this.MIQAConfig.artifact_options.map((name) => ({
+        value: name,
+        labelText: this.convertValueToLabel(name),
+      }));
+    },
+    chips() {
+      return this.artifacts.map((artifact) => [artifact, this.getCurrentChipState(artifact)]);
+    },
+    suggestedArtifacts() {
+      if (this.currentViewData.scanDecisions.length > 0) {
+        const lastDecisionArtifacts = _.last(_.sortBy(
+          this.currentViewData.scanDecisions, (dec) => dec.created,
+        )).user_identified_artifacts;
+        return Object.entries(lastDecisionArtifacts).filter(
+          ([, present]) => present === 1,
```

Comment:
a string enum on the backend would translate to a TS type with an enum field, which would remove this magic number.